docs/standard/exceptions/best-practices-for-exceptions.md

```diff
@@ -9,20 +9,16 @@ dev_langs:
 helpviewer_keywords: 
   - "exceptions, best practices"
 ms.assetid: f06da765-235b-427a-bfb6-47cd219af539
-author: "mairaw"
-ms.author: "mairaw"
 ---
 # Best practices for exceptions
 
 A well-designed app handles exceptions and errors to prevent app crashes. This section describes best practices for handling and creating exceptions.
 
-## Use try/catch/finally blocks
+## Use try/catch/finally blocks to recover or clean resources
 
-Use `try`/`catch`/`finally` blocks around code that can potentially generate an exception. 
+Use `try`/`catch` blocks around code that can potentially generate an exception ***and*** your code can recover from that exception. In `catch` blocks, always order exceptions from the most specific to the least specific. When your code cannot recover from an exception, don't catch that exception. Enable methods further up the call stack to recover if possible.
 
-In `catch` blocks, always order exceptions from the most specific to the least specific.
-
-Use a `finally` block to clean up resources, whether you can recover or not.
+Clean up resources allocated with either `using` statements, or `try`/`finally` blocks. Prefer `using` statements to automatically clean up resources when exceptions are thrown. Use  `try`/`finally` blocks to clean up resources that don't implement <xref:System.IDisposable>.
```

Comment:
isn't it preferable to describe this as simply a finally block, rather than a try/finally block? And perhaps note that the finally block is (generally) always executed.